index.js

```diff
@@ -1,21 +1,3 @@
-/* global Blob, FileReader */
+/* global Response, Buffer */
 
-module.exports = function blobToBuffer (blob, cb) {
-  if (typeof Blob === 'undefined' || !(blob instanceof Blob)) {
-    throw new Error('first argument must be a Blob')
-  }
-  if (typeof cb !== 'function') {
-    throw new Error('second argument must be a function')
-  }
```

Comment:
Can we keep this check in, but reverse the condition? It would be nice to help folks who are upgrading from 1.x and might leave the callback in by accident? It's similar to how we left the check in `stream-to-blob` when switching to promises: https://github.com/feross/stream-to-blob/blob/34d257b46d0f8c5d710194336d87f0623aa60fd4/index.js#L7

Reply:
mm, don't follow... 😕 

maybe close this PR just and let you do as you want?